NOTICE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2013-Present CloudFoundry.org Foundation, Inc. All Rights Reserved.
```

Comment:
Not sure this is correct?

Reply:
took that from here:

https://github.com/ninech/buildpack-rails-permissions/blob/main/NOTICE

Reply:
Yeah, not sure if it is correct there either :-)

Reply:
removed it 👍

Reply:
will also remove it there